UnitsNet/UnitDefinitions/Length.json

```diff
@@ -130,6 +130,38 @@
       ]
     },
     {
+      "SingularName": "Fathom",
+      "PluralName": "Fathoms",
+      "FromUnitToBaseFunc": "x*1.8288",
+      "FromBaseToUnitFunc": "x/1.8288",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "fathom" ]
+        },
+        {
+          "Culture": "ru-RU",
```

Comment:
You should omit the entire array element for the Russian abbreviation if you don't have one, which is fine.
It will always fallback to the US English for cultures that don't have an abbreviation defined.